### PR TITLE
chore(docs): improves URL macros in README.md

### PR DESCRIPTION
**Pull request by pranshuchittora**
Improved URL macros in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,61 +1,56 @@
...
 
 Now that we've covered much of the backlog of _missing documentation_, we are
 starting to heavily review each section of the site's content to sort and
 structure it appropriately. The following issues should provide a pretty good
 idea of where things are, and where they are going:
 
-- [Guides - Review and Simplify][1]
-- [Concepts - Review and Organize][2]
-- [API - v4 Rewrite][8]
+- [Guides - Review and Simplify][issue-1258]
```
Comment (montogeek) [inline README.md]: Why use numbers instead of `guides`, `concepts`, `api` like you did in the rest of the references?

Comment (pranshuchittora) [reply README.md]: I was not sure between this & issue number.
Will change it :+1:

Comment (pranshuchittora) [reply README.md]: @montogeek pls review the new changes.

Thank you

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**